lapack/testlapack/dgesv.go

```diff
@@ -0,0 +1,158 @@
+// Copyright ©2021 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testlapack
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"golang.org/x/exp/rand"
```

Comment:
Blank between.

Reply:
Thanks, done. I've also added a check of `ok` in the test.